_posts/newsletters/2021-05-22-newsletter.md

```diff
@@ -0,0 +1,115 @@
+# **Upcoming CFP: RSE-HPC-2021 workshop at SC21**
+
+[Research Software Engineers in HPC (RSE-HPC-2021)](https://us-rse.org/rse-hpc-2021/) has been accepted as a workshop at [SC21](https://sc21.supercomputing.org/) in November!  The Call for Papers will go out in early June, with a submission deadline in early August.  Details of the call are being finalized and will be announced soon.
+
+<a name="jobs"></a>
+# **Recent Job Postings**
+
+These opportunities were recently posted to the [RSE Careers page](https://us-rse.org/jobs/):
+1. [Senior Machine Learning in Medical Imaging Research Software & Systems Engineer](https://eng.ox.ac.uk/jobs/job-detail/?vacancyID=151088): Oxford Biomedical Image Analysis Laboratory, Department of Engineering Science, University of Oxford, UK
+2. [Neutron Data and Computing Program Manager](https://jobs.ornl.gov/job/Oak-Ridge-Neutron-Data-and-Computing-Program-Manager-TN-37830/724194700/): Oak Ridge National Laboratory, Oak Ridge, TN
+3. [Senior Research Software Engineer](https://main-princeton.icims.com/jobs/12582/senior-research-software-engineer/job): Princeton University, Princeton, NJ
+4. [Research Software Engineer](https://brown.wd5.myworkdayjobs.com/en-US/staff-careers-brown/job/180-George-Street/Research-Software-Engineer_REQ169311): Brown University, Providence, RI
+5. [HPC Applications Performance Engineer, Hewlett Packard Enterprise](https://hpe.wd5.myworkdayjobs.com/Jobsathpe/job/Bloomington-Minnesota-United-States-of-America/HPC-APPLICATIONS-PERFORMANCE-ENGINEER_1080055-1): Various locations
+6. [Software Engineer](https://recruit.ap.ucsb.edu/JPF01941): Benioff Ocean Initiative at University of California, Santa Barbara, CA (currently remote)
+
+
+<a name="news"></a>
+# **Community News**
+
+* Daniel S. Katz and Neil Chue Hong are editing a special issue of PeerJ CS on software citation, indexing and discoverability, so if you are doing any work in this area, please consider submitting it to this [special issue](https://peerj.com/special-issues/84-software).
+* DHTech, a community of people doing technical work in Digital Humanities, are conducting a survey to gauge the interest for a technical mentorship program. If you work in Digital Humanities, please consider filling out the survey! More information can be found [on the DHTech website](https://dh-tech.github.io/mentorship-survey).
+
+<a name="reads"></a>
+# **Interesting Reads**
+
+Items you may have missed on the blog and Slack:
+* The 10 Best Practices for Remote Software Engineering:  [https://cacm.acm.org/opinion/articles/252174-the-10-best-practices-for-remote-software-engineering/fulltext](https://cacm.acm.org/opinion/articles/252174-the-10-best-practices-for-remote-software-engineering/fulltext)
+* The Institute for Research and Innovation in Software for High Energy Physics (IRIS-HEP) published an article on [building a career path for research software engineers](https://iris-hep.org/2021/05/12/career-path-rse.html).
+
```

Comment:
* Implementing FAIR for research software: attitudes, advantages and challenges, [part one](https://software.ac.uk/blog/2021-05-19-implementing-fair-research-software-attitudes-advantages-and-challenges-part-one) and [part two](https://www.software.ac.uk/blog/2021-05-20-implementing-fair-research-software-attitudes-advantages-and-challenges-part-two?_ga=2.132757834.599148255.1621692340-781456882.1621522583), by Leyla Jael Castro, Diego Alonso Álvarez, Stian Soiland-Reyes, Markus Löning, Sam Harrison, Faruk Diblen, Abhishek Dasgupta, Morane Gruenpeter and Paula Andrea Martinez, from the Software Sustainability Institute's Collaborations Workshop 2021 speed blog series.

Reply:
```suggestion
* Implementing FAIR for research software: attitudes, advantages and challenges, [part one](https://software.ac.uk/blog/2021-05-19-implementing-fair-research-software-attitudes-advantages-and-challenges-part-one) and [part two](https://www.software.ac.uk/blog/2021-05-20-implementing-fair-research-software-attitudes-advantages-and-challenges-part-two), by Leyla Jael Castro, Diego Alonso Álvarez, Stian Soiland-Reyes, Markus Löning, Sam Harrison, Faruk Diblen, Abhishek Dasgupta, Morane Gruenpeter and Paula Andrea Martinez, from the Software Sustainability Institute's Collaborations Workshop 2021 [speed blog series](https://www.software.ac.uk/tags/cw21-speed-blog-posts).
```